lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -429,7 +441,8 @@ def replicate_db_instance(module, conn):
             instance = RDSDBInstance(response['DBInstance'])
             changed = True
         except botocore.exceptions.ClientError as e:
-            module.fail_json(msg="Failed to create replica instance: %s " % str(e))
+            module.fail_json(msg="Failed to create replica instance: %s " % str(e),
+                             exception=traceback.format_exc() )
```

Comment:
boto3 exception guidelines also say to add `**camel_dict_to_snake_dict(e.response)`